README.md

```diff
@@ -43,11 +43,11 @@ You can supply the tokens using command line arguments:
 
 You can also supply log, stats and event tokens using environment variables.  
 
-When both command line arguments and environment variables are supplied, the environment variables are used.
+When both command line arguments and environment variables are supplied, the command line arguments are used.
 - `INSIGHT_LOGSTOKEN`: Log token for logs
 - `INSIGHT_STATSTOKEN`: Log token for stats
 - `INSIGHT_EVENTSTOKEN`: Log token for events
-- `INSIGHT_TOKEN`: Log token which is used for the above tokens if one is not provided.
+- `INSIGHT_TOKEN`: Log token which is used for any of the above tokens if one is not provided.
```

Comment:
so this is the fallback, with the above 3 being overrides for specific types?

Reply:
Yes, you can specify which log to send logs/stats/events specifically.
If you don't specify one of them then the main token supplied will be used for each.
If you don't supply either we crash